From c68ca316e6462504277c49ecd36b23449cb695aa Mon Sep 17 00:00:00 2001
From: Pierre Moreau <pierre.moreau@cs.lth.se>
Date: Mon, 17 Aug 2020 16:35:47 +0200
Subject: [PATCH] Fix GLM version

Given the amount of changes going in between revision versions, like
CMake support being removed in 0.9.9.6 and then later re-introduced in
0.9.9.7, it is important to be able to compare the full version of GLM
including its revision number.

Signed-off-by: Pierre Moreau <pierre.moreau@cs.lth.se>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 10bb1af..1a49412 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
 cmake_policy(VERSION 3.2)
 
-set(GLM_VERSION "0.9.9")
+set(GLM_VERSION "0.9.9.5")
 project(glm VERSION ${GLM_VERSION} LANGUAGES CXX)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
-- 
2.28.0

